cmd/api/src/model/appcfg/parameter.go

```diff
@@ -283,3 +285,21 @@ func GetReconciliationParameter(ctx context.Context, service ParameterService) b
 
 	return result.Enabled
 }
+
+type ScheduledAnalysisParameter struct {
+	Enabled bool      `json:"enabled,omitempty"`
+	RRule   string    `json:"rrule,omitempty"`
+	LastRun time.Time `json:"last_run,omitempty"`
+}
+
+func GetScheduledAnalysisParameter(ctx context.Context, service ParameterService) (ScheduledAnalysisParameter, error) {
+	result := ScheduledAnalysisParameter{Enabled: false, RRule: "", LastRun: time.Time{}}
+
+	if cfg, err := service.GetConfigurationParameter(ctx, ScheduledAnalysis); err != nil {
+		return result, err
+	} else if err := cfg.Map(&result); err != nil {
+		return result, err
+	}
+
+	return result, nil
```

Comment:
For the parameters service, the existing parameters all fallback to default values and don't return errors. Would it be possible to keep that consistency here as well?

Reply:
I'm actually not sure if that behavior works here, since if we have a db error, the returned object would have `enabled:false` which would cause analysis to trigger.  In this particular case, I think its important that we get the correct value instead of a fallback